packages/edit-site/src/components/page-templates/dataviews-templates.js

```diff
@@ -153,6 +195,22 @@ export default function DataviewsTemplates() {
 	}, [ allTemplates, view ] );
 	const fields = useMemo(
 		() => [
+			{
+				header: __( 'Preview' ),
+				id: 'preview',
+				getValue: () => {},
+				render: ( { item, view: { type: viewType } } ) => {
+					return (
+						<TemplatePreview
+							content={ item.content.raw }
+							viewType={ viewType }
+						/>
+					);
+				},
+				minWidth: 120,
```

Comment:
I mentioned elsewhere, but this data is only for one of the layouts (list). We should find a different way to provide this. Either by making it explicity in the field API (e.g.: set in under `layout.list.minWidth`) or by absorving it in the view config (in principle, I'd prefer this option, as it relates to control layout).

Not to fix in this PR, but important to figure out before stabilizing/creating a package for dataviews.

Reply:
Agreed, but let's explore this separately. Probably it should be enough by just adding specific css on each field, per view type.